### ISSUE TITLE
deletion of task has no effect

### ISSUE THREAD

**Issue by haegar33** [Doist/todoist-api-python#83]
## Bug description
using the api.delete_task() function reports success but task are still present

## Expected behaviour
when executing the api.delete_task() twice I would expect an error for the second call as task should be already deleted

## Is reproducible
Yes

## To reproduce

Code Snipplet:
```
from todoist_api_python.api import TodoistAPI
api = TodoistAPI(key)
```

I receive a list of completed task via the REST API as this does not work with the python module. I get a python dict _clist_, then:

```
  for c in clist:
        print('Delete item: ',c['content'],c['task_id'])
        try:
            is_success = api.delete_task(task_id=c['task_id'])
            print(is_success)
        except Exception as error:
            print(error)
```

works fine, but when I run the script a second time **all "deleted" tasks are back again**

## Version information:
- Package version: 8.1.3
- Python version: 3.10

**Comment by haegar33** [comment]
actually the problem is not for the python module but for all API calls! Whether using the Sync V9 or REST V2 API it is not possible to delete tasks. Looks like this intentional by the company to preserve your data for commercial reasons?

**Comment by goncalossilva** [comment]
Indeed, deleting tasks through the API _should_ work. If there is a case of a specific existing task where it doesn't, I'd appreciate it if you could [report it to our support team](https://www.todoist.com/contact), and include the ID! We'd definitely take a look. It's worth noting our official apps use the API as well, so if it works there, it likely works here as well.

As for subsequent deletions not raising an error, deletions are (by design) idempotent in Todoist's API. So attempting to delete a previously deleted task will work. This is intended.
